api/src/main/resources/marquez/db/migration/V20__alter_jobs_to_add_namespace_name.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE jobs ADD namespace_name VARCHAR(255);
```

Comment:
With []() now merged, mind updating the migration version to `V22`?

Reply:
pr #935 was branched off of this commit so this pr can be closed

Reply:
👍